WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -29,12 +29,10 @@ public partial class CoinJoinStateViewModel : ViewModelBase
 	private readonly MusicStatusMessageViewModel _roundSucceedMessage = new() { Message = "Successful coinjoin, continuing..." };
 	private readonly MusicStatusMessageViewModel _roundFinishedMessage = new() { Message = "Round finished, waiting for next round" };
 	private readonly MusicStatusMessageViewModel _abortedNotEnoughAlicesMessage = new() { Message = "Not enough participants, retrying..." };
-	private readonly MusicStatusMessageViewModel _outputRegistrationMessage = new() { Message = "Constructing coinjoin" };
+	private readonly MusicStatusMessageViewModel _coinJoinInProgress = new() { Message = "Coinjoin in progress" };
 	private readonly MusicStatusMessageViewModel _inputRegistrationMessage = new() { Message = "Waiting for other participants" };
-	private readonly MusicStatusMessageViewModel _transactionSigningMessage = new() { Message = "Finalizing coinjoin" };
 	private readonly MusicStatusMessageViewModel _waitingForBlameRoundMessage = new() { Message = "Waiting for the fallback round" };
```

Comment:
Maybe this one should be deleted also and message set to "Coinjoin in progress" all the time.
"fallback round" is imo not understandable as is for most of users. The word is shady but the reality simple, we're just retrying with excluding some guys. Coinjoin never stopped to be in progress, it makes sense.

Reply:
Yeah good point, the user doesn't really care about if a round failed, but since they will participate in a new round, which would cause the reset of the timer, a transforming "state" might be a good option.

Anyway, I will think about it, thanks for pointing it out.